content/webapp/pages/article.tsx

```diff
@@ -225,7 +229,16 @@ const ArticlePage: FC<Props> = ({ article, jsonLd }) => {
                   </Space>
                 </Fragment>
               ))}
-
+            {readingTime && article.readingTime ? (
+              <Fragment>
+                <span className={font('intr', 6)}>
+                  <span className={font('intr', 5)}>| </span> reading time{' '}
+                  <span className={font('intb', 6)}>
+                    {readingTimeInMinutes} min
```

Comment:
nit: you seem to use/not use this variable interchangeable (see L232), I’d suggest ditching it and just using `article.readingTime` directly.

I also wonder if having `min` applied here is right (because now the knowledge that this is minutes has to cross a file boundary), or whether it'd be better to dump a string here `8 min`, where the `min` is applied on the server.

Reply:
Good point on both counts. I've adjusted things to account for that